src/types.ts

```diff
@@ -63,8 +63,9 @@ export interface Schema {
       questions: Array<{
         id: string;
         label?: string;
-        value?:string;
-        type: string;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Rather than adding this as a comment, lets follow what has been done in the form engine and add this as a rule to the eslintrc file. See https://github.com/openmrs/openmrs-esm-form-engine-lib/blob/7f9a0813b1c69450e2cc0fcfa5222cd880b6033d/.eslintrc#L16

Reply:
Thanks, @NethmiRodrigo! Adding it directly to the `.eslintrc` file, like in the form engine, was a great tip—appreciate the help!

Reply:
Actually, we would prefer that the no-explicit-any rule is _enabled_ and disclaimed like this where it's unavoidable (although, for preference, use `unknown` rather than `any`).

Reply:
FWIW, the only reason its currently disabled in other repos is because historically we didn't enforce it.